composer.json

```
@@ -69,6 +69,7 @@
         "prestashop/graphnvd3": "^2",
         "prestashop/gridhtml": "^2",
         "prestashop/gsitemap": "^4",
+        "prestashop/hummingbird": "dev-develop#76f0070a140de899029036ad645aa08594a480d0",
```

```suggestion
        "prestashop/hummingbird": "dev-develop",
```
You don't need to specify the commit here, only the branch is needed The commit is then saved in the lock file every time we run `composer update` again

---

BTW even the CI doesn't like it https://github.com/PrestaShop/PrestaShop/actions/runs/9496040516/job/26169578438?pr=36198 😅

---

I know that but I want to launch CI with this pending PR https://github.com/PrestaShop/hummingbird/pull/629